hikari/templates.py

```diff
@@ -31,6 +31,8 @@
 import attr
 
 from hikari import guilds
+from hikari import traits
+from hikari import undefined
```

Comment:
These can be moved into the ```
if typehint.TYPE_CHECKING:
```
block, as they are not required at runtime

Reply:
Done! <3